backend/app-orchestrator-service/api/src/main/kotlin/api/Compute.kt

```diff
@@ -89,6 +89,23 @@ data class ComputeOpenInteractiveSessionRequestItem(
 )
 data class ComputeOpenInteractiveSessionResponse(val sessions: List<OpenSession>)
 
+typealias UtilizationRequest = Unit
+data class UtilizationResponse(
+    val allocatable: CpuAndMemory,
+    val used: CpuAndMemory,
+    val jobs: JobUtilization
+)
```

Comment:
Doesn't follow new naming convention: https://github.com/SDU-eScience/UCloud/issues/1934

```suggestion
typealias ComputeUtilizationRequest = Unit
data class ComputeUtilizationResponse(
    val allocatable: CpuAndMemory,
    val used: CpuAndMemory,
    val jobs: JobUtilization
)

```